apps/azure-functions-api/src/utils/health-check.ts

```diff
@@ -0,0 +1,7 @@
+import { HealthCheck } from "@pagopa/io-functions-commons/dist/src/utils/healthcheck";
```

Comment:
this kind of things must go into a standard (non IO specific) azure function typescript library, I'd remove any reference to `io-functions-commons` until we got that

Reply:
done in d029f98